src/components/ColumnRenderHelper.tsx

```diff
@@ -24,11 +24,13 @@ export function renderPenetrance(cellProps: any)
 export function renderHgvsg(cellProps: any)
 {
     const constructLink = (hgvsg: string, content: JSX.Element) => <Link to={`/variant/${hgvsg}`}>{content}</Link>;
```

Comment:
Just realized we can also simply do this :)
```suggestion
    const constructLink = (hgvsg: string) => <Link to={`/variant/${hgvsg}`}>{hgvsg}</Link>;
```

Reply:
lgtm